extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php

```diff
@@ -38,14 +38,12 @@ public function getSourceLabel() {
 		if ( count( $sources ) > 1 ) {
 			foreach ( $sources as $source ) {
 				if ( !empty( $source ) ) {
-					$sourceLabel = !empty( $label ) ? "{$label} ({$source})" : $label;
-					$sourceLabels = array_merge( $sourceLabels, [ $source => $sourceLabel ] );
+					$sourceLabel = !empty( $label ) ? "{$label} ({$source})" : '';
+					$sourceLabels[$source] = $sourceLabel;
```

Comment:
You are sure you don't want to make it a one line? ``$sourceLabel`` is assignment is unnecessary here.